### PR TITLE
🍰 Add Wallet Link To Mails

### LINKED ISSUE #1734: 🚀 [Feature] Mails must contain link to wallet
## 🚀 Feature

Mails must contain link to wallet since its the first thing a User wants to do when receiving GDD - check his account.

- [x] adjust `Receive GDD` EMail
- [x] check other EMails if it would make sense to include link to wallet

### PR DESCRIPTION
**Pull request by Tirokk**
### Issues

- fixes #1734 

### Todo

- [x] should other e-mails containing this link as well? or other links?
  - no
- [x] set correct client URI
- [x] implement `EMAIL_CLIENT_URI` config
  - [x] therefore increment `CONFIG_VERSION` by searching for `v4.2022-04-05`
- [x] test e-mail text and link manually
- [x] write tests
- [x] clean up: search for `Wolle`

### CODE REVIEW

File: backend/src/graphql/resolver/TransactionResolver.ts [hunk 1]
```diff
@@ -125,6 +127,7 @@ export const executeTransaction = async (
   }
 
   // send notification email
+  const overviewURL = new URL('/overview', CONFIG.EMAIL_CLIENT_URI)
```
Comment (ulfgebhardt) [inline backend/src/graphql/resolver/TransactionResolver.ts:130]: please specify the full URL in the config parameter.
Consider to name the config value properly - its not an email client, but the `EMAIL_WALLET_URI`

Comment (Tirokk) [reply backend/src/graphql/resolver/TransactionResolver.ts:129]: I like to discuss this with you personally @ulfgebhardt .

File: backend/src/mailer/text/transactionReceived.ts [hunk 2]
```diff
@@ -25,6 +26,9 @@ ${data.memo}
 Bitte antworte nicht auf diese E-Mail!
 
 Mit freundlichen Grüßen,
-dein Gradido-Team`,
+dein Gradido-Team
+
+
+Link zur Übersicht: <a href="${data.overviewURL}">Deine Übersicht</a>`,
```
Comment (Tirokk) [inline backend/src/mailer/text/transactionReceived.ts:32]: Is the design approbriate?

Comment (ulfgebhardt) [reply backend/src/mailer/text/transactionReceived.ts:32]: Actually no, since its a text email - no html tags here

Comment (Tirokk) [reply backend/src/mailer/text/transactionReceived.ts:32]: How I can get a link in the text then?

**Comment by Mogge** [comment]
What do you think about this package?
https://github.com/forwardemail/email-templates

**Comment by Tirokk** [comment]
> https://github.com/forwardemail/email-templates

We can investigate this for the future. I can make an issue for it? @Mogge 
👍🏼

But in this issue just adding a link is the task I do understand.

**Comment by Tirokk** [comment]
If I pull the actual master then the frontend tests are not running anymore @ogerly

File: backend/src/mailer/text/transactionReceived.ts (hunk 2 above)
Comment (Tirokk) [reply backend/src/mailer/text/transactionReceived.ts]: I removed the HTML.

File: backend/src/graphql/resolver/TransactionResolver.ts (hunk 1 above)
Comment (Tirokk) [reply backend/src/graphql/resolver/TransactionResolver.ts]: Ulf explaind me his reasons and I followed his suggestions.

File: backend/src/mailer/text/transactionReceived.ts [hunk 3]
```diff
@@ -25,6 +26,9 @@ ${data.memo}
 Bitte antworte nicht auf diese E-Mail!
 
 Mit freundlichen Grüßen,
-dein Gradido-Team`,
+dein Gradido-Team
+
+
+Link zu deiner Übersicht: ${data.overviewURL}`,
```
Comment (ulfgebhardt) [inline backend/src/mailer/text/transactionReceived.ts:32]:
```suggestion
Link zu deinem Konto: ${data.overviewURL}`,
```

**Review by ulfgebhardt — COMMENTED** [review]
This looks very good - I have one more textual change request and the change for `deployment/bare_metal/.env.dist` is missing:
- new config version
- overview value for stage1.gradido.net

**Review by Mogge — APPROVED** [review]
Welcome aboard!

File: backend/src/mailer/text/transactionReceived.ts (hunk 3 above)
Comment (Tirokk) [reply backend/src/mailer/text/transactionReceived.ts:32]: As far I can see the changes in `deployment/bare_metal/.env.dist` are there:

- https://github.com/gradido/gradido/blob/942e39534b84dc420c7ee3ff5c9fdbe80368bfcb/deployment/bare_metal/.env.dist#L21
- https://github.com/gradido/gradido/blob/942e39534b84dc420c7ee3ff5c9fdbe80368bfcb/deployment/bare_metal/.env.dist#L47

**Review by Mogge — APPROVED** [review]

**Pull request merged**